src/renderer/contexts/ExecutionContext.tsx

```diff
@@ -389,59 +388,57 @@ export const ExecutionProvider = memo(({ children }: React.PropsWithChildren<{}>
         setIteratorProgress({});
     }, [backend, restart, sendAlert]);
 
-    useHotkeys(
-        'F5',
-        useCallback(() => {
-            switch (status) {
-                case ExecutionStatus.READY:
-                case ExecutionStatus.PAUSED:
-                    // eslint-disable-next-line @typescript-eslint/no-floating-promises
-                    run();
-                    break;
-                case ExecutionStatus.RUNNING:
-                case ExecutionStatus.KILLING:
-                    break;
-                default:
-                    assertNever(status);
+    // This makes sure keystrokes are executed even if the focus is on an input field
+    useEffect(() => {
+        const handleKeyPress = (event: KeyboardEvent) => {
+            if (event.code === 'F5') {
+                switch (status) {
+                    case ExecutionStatus.READY:
+                    case ExecutionStatus.PAUSED:
+                        event.preventDefault();
+                        (run as () => void)();
```

Comment:
Please don't use type casts like this. If we ever change the signature of those functions, TS will not be able to catch the now incorrect call.

You likely did this because ESLint doesn't like floating promises. The easiest way around this is to do what ESLint wants and add a `catch`.
```suggestion
                        run().catch(log.error);
```

Reply:
done